python/cuml/test/test_incremental_pca.py

```diff
@@ -109,3 +110,17 @@ def test_partial_fit(nrows, ncols, n_components, density,
 
     assert array_equal(cu_inv, sk_inv,
                        5e-5, with_sign=True)
+
+def test_exceptions():
+    with pytest.raises(TypeError):
+        X = cupyx.scipy.sparse.random(10, 10)
```

Comment:
These don't need to be random numbers, they can just be arrays of 1's or 0's no? Not a big deal given the size, but should be slightly faster

Reply:
Also `X` can be initialized only once outside the `with pytest.raises..` contexts